src/test/test_customvision.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import os
+from customvision.classifier import Classifier
+
+from test.test_api import construct_path
+from test import config as cfg
+
+
+@pytest.fixture
+def classifier():
+    """
+        initialize custom vision classifer object
+    """
+    yield Classifier()
+
+
+def test_prediction_image_does_not_crash(classifier):
+    """
+        assert classifier is able to get a predicition without crashing
+    """
+    path = construct_path(cfg.api_path_data)
+    path = os.path.join(path, cfg.cv_test_image)
+    with open(path, "rb") as fh:
+        try:
+            best_guess, probabilitites = classifier.predict_image(fh)
+        except Exception:
+            assert False
+        assert True
+
+
+def test_best_guess_is_string(classifier):
+
+    path = construct_path(cfg.api_path_data)
+    path = os.path.join(path, cfg.cv_test_image)
+    with open(path, "rb") as fh:
+        best_guess, probabilitites = classifier.predict_image(fh)
+
```

Comment:
Why is there a newline here? If possible, the assert should be done after 'with open' (meaning it's not indented). In this case a newline is reasonable to use.